### PR TITLE
Fixed existing detekt/lint issues

### PR DESCRIPTION
**Pull request by LachlanMcKee**
## Description
Fixed the majority of the outstanding detekt/lint errors.
The remaining issues should be investigated further.

## Check list

- [x] I have updated `CHANGELOG.md` if required.
- [x] I have updated documentation if required.

### CODE REVIEW

File: testing-ui/src/main/java/com/bumble/appyx/testing/ui/rules/AppyxTestRule.kt [hunk 1]
```diff
@@ -16,6 +16,7 @@ open class AppyxTestRule<T : Node>(
     private val launchActivity: Boolean = true,
     private val composeTestRule: ComposeTestRule = createEmptyComposeRule(),
     /** Add decorations like custom theme or CompositionLocalProvider. Do not forget to invoke `content()`. */
+    @Suppress("unused")
```
Comment (LachlanMcKee) [inline testing-ui/src/main/java/com/bumble/appyx/testing/ui/rules/AppyxTestRule.kt:19]: unsure about whether to remove this variable or not

Comment (CherryPerry) [reply testing-ui/src/main/java/com/bumble/appyx/testing/ui/rules/AppyxTestRule.kt:19]: Keep it and create an issue, I forgot to use it.

File: sandbox/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,3 +1,6 @@
 <resources>
     <string name="app_name">Appyx sandbox</string>
+    <string name="rib_root_title">Black area belongs to 1.0, container below to 2.0</string>
```
Comment (CherryPerry) [inline sandbox/src/main/res/values/strings.xml:3]: I think you could just add `HardcodedText` to Lint ignore list for sandbox/app projects.

File: plugins/verification-plugin/src/main/kotlin/DetektPlugin.kt [hunk 3]
```diff
@@ -13,7 +13,14 @@ class DetektPlugin : Plugin<Project> {
             target.extensions.configure<DetektExtension> {
                 buildUponDefaultConfig = true
                 baseline = target.file("detekt-baseline.xml")
-                config.from(target.rootProject.file("detekt.yml"))
+
+                val localDetektConfig = target.file("detekt.yml")
+                val rootDetektConfig = target.rootProject.file("detekt.yml")
```
Comment (CherryPerry) [inline plugins/verification-plugin/src/main/kotlin/DetektPlugin.kt:18]: Does specifying both files always instead of checking existence work?

**Review by CherryPerry — APPROVED** [review]

File: sandbox/src/main/res/values/strings.xml (hunk 2 above)
Comment (LachlanMcKee) [reply sandbox/src/main/res/values/strings.xml:3]: done

File: plugins/verification-plugin/src/main/kotlin/DetektPlugin.kt (hunk 3 above)
Comment (LachlanMcKee) [reply plugins/verification-plugin/src/main/kotlin/DetektPlugin.kt]: looks like that works. good idea!

Comment (LachlanMcKee) [reply plugins/verification-plugin/src/main/kotlin/DetektPlugin.kt:18]: scratch that... I didn't run it correctly locally when testing. reverted back to how it was.

**Pull request merged**